iroha_actor/Cargo.toml

```diff
@@ -0,0 +1,20 @@
+[package]
+name = "iroha_actor"
+version = "0.1.0"
+authors = ["i1i1"]
+edition = "2018"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[features]
+default = ["deadlock_detection"]
```

Comment:
I don't think `deadlock_detection` should be in default features, especially if we don't use it with broker.